src/js/components/AppListComponent.jsx

```diff
@@ -186,7 +187,10 @@ var AppListComponent = React.createClass({
 
     if (filterText != null && filterText !== "") {
       nodesSequence = nodesSequence
-        .filter(app => app.id.indexOf(filterText) !== -1);
+        .filter(app => {
+          return score(app.id, filterText) > 0.025;
```

Comment:
The score value is our way of determining matches in the app list for the given filterText. https://github.com/atom/fuzzaldrin#scorestring-query 

If we move back to indexOf we lose the whole fuzz matching algorithm completely! Why would you want to do that? I agree that the result not showing up was a regression, but that's because i set the score threshold too high. Please try again now.